js-intro.js

```diff
@@ -4,58 +4,195 @@
 // Please read all questions thoroughly
 // Psuedo coding is highly recommended
 
+
+//-------------------------------------------------------------------------
+
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
+
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+
+/*
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
+// Creating boiling point temp
+var tempBoil = 212
+
+//Creating a series of functions for each temp
```

Comment:
How can you create a single function that takes in _any_ temperature as an integer, and returns if it is below, at, or above boiling point? 
HINT **`else if` decision trees.
I do like the variable naming here, however `butseriouslyisit`acts as a placeholder for outside information to be used later in your function. Rather than referring back to `butseriouslyisit`, you have `temp1` 2 & 3, all hard-coded into your functions, essentially rendering the use of an argument useless. Let me know if you need me to clarify anything and I'd be happy to help :)

Reply:
Understood! Thank you!